api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -470,6 +480,165 @@ function rewardReducer(network, validators, reward, reducers) {
+    type: proposalTypeEnum.TREASURY,
+    title: `Treasury Proposal #${proposal.id}`,
+    creationTime: proposal.creationTime,
+    status: `VotingPeriod`,
+    statusEndTime: getStatusEndTime(blockHeight, proposal.council[0].votes.end),
+    tally: councilTallyReducer(proposal.council[0].votes, councilMembers),
+    deposit: toViewDenom(
+      network,
+      Number(proposal.proposal.bond),
+      network.stakingDenom
+    ),
+    proposer: proposal.proposal.proposer.toHuman(),
+    beneficiary: proposal.proposal.beneficiary // the account getting the tip
+  }
+}
+
+function councilProposalReducer(
+  network,
+  proposal,
+  councilMembers,
+  blockHeight
+) {
+  return {
+    id: `Council-`.concat(proposal.votes.index),
+    networkId: network.id,
+    type: proposalTypeEnum.COUNCIL,
+    title: `Council Proposal #${proposal.votes.index}`,
+    description: proposal.description,
+    creationTime: proposal.creationTime,
+    status: `VotingPeriod`,
+    statusBeginTime: proposal.creationTime,
+    statusEndTime: getStatusEndTime(blockHeight, proposal.votes.end),
+    tally: councilTallyReducer(proposal.votes, councilMembers),
+    deposit: undefined,
+    proposer: undefined
+  }
+}
+
+function tallyReducer(network, tally, totalIssuance) {
+  //
+  // tally chain format:
+  //
+  // "tally": {
+  //   "ayes": "0x0000000000000000001e470441298100",
+  //   "nays": "0x00000000000000000186de726fc56000",
+  //   "turnout": "0x000000000000000000dc3dd9ad3d0800"
+  // }
+  //
+  // turnout is the real amount deposited by voters
+  // in polkadot you can vote with "conviction", that means
+  // ayes and nays are amplified by the selected lockup period:
+  //
+  // 1x voting balance, locked for 1x enactment (8.00 days)
+  // 2x voting balance, locked for 2x enactment (16.00 days)
+  // 3x voting balance, locked for 4x enactment (32.00 days)
+  // 4x voting balance, locked for 8x enactment (64.00 days)
+  // 5x voting balance, locked for 16x enactment (128.00 days)
+  // 6x voting balance, locked for 32x enactment (256.00 days)
+  //
+
+  const turnout = BigNumber(tally.turnout)
+
+  const totalVoted = BigNumber(tally.ayes).plus(tally.nays)
+  const total = toViewDenom(network, totalVoted.toString(10))
+  const yes = toViewDenom(network, tally.ayes)
+  const no = toViewDenom(network, tally.nays)
+  const totalVotedPercentage = turnout
+    .div(BigNumber(totalIssuance))
+    .toNumber()
+    .toFixed(4) // the percent conversion is done in the FE. We just send the decimals here
+
+  return {
+    yes,
+    no,
+    abstain: 0,
+    veto: 0,
+    total,
+    totalVotedPercentage
+  }
+}
+
+function councilTallyReducer(votes, councilMembers) {
+  const total = votes.ayes.length + votes.nays.length
+  return {
+    yes: votes.ayes.length,
+    no: votes.nays.length,
+    abstain: 0,
+    veto: 0,
+    total,
+    totalVotedPercentage: (total / councilMembers.length).toFixed(4) // the percent conversion is done in the FE. No need to multiply by 100
```

Comment:
each council member has the same voting power? who is voting on council members?

Reply:
oh, I see what you mean. True, it is not the real number

Reply:
OK, done. Although I see another thing coming here:

```
Prime Members
The council, being an instantiation of Substrate's Collective pallet, implements what's called a prime member whose vote acts as the default for other members that fail to vote before the timeout.

The prime member is chosen based on a Borda count.

The purpose of having a prime member of the council is to ensure a quorum, even when several members abstain from a vote. Council members might be tempted to vote a "soft rejection" or a "soft approval" by not voting and letting the others vote. With the existence of a prime member, it forces councillors to be explicit in their votes or have their vote counted for whatever is voted on by the prime.
```

Reply:
But this can be calculated once the voting period is closed :thinking: (the impact of the council prime member, I mean)

Reply:
This seems very advanced though. Let's do it afterwards. I just left a TODO comment

Reply:
yep let's leave it for now!